geniza/corpus/templates/corpus/document_list.html

```diff
@@ -31,12 +29,14 @@ <h1>{{ doctitle }}</h1>
     {% for document in documents %}
         <li class="result">
 
+            {% if document.tags.exists %}
```

Comment:
does this work? tags is a list in a solr result, not a queryset (IIRC)

Reply:
it appeared to...but I'll switch to just `document.tags`